main.go

```diff
@@ -114,20 +115,29 @@ func main() {
 			if (*flagMaxFails > 0) && (fails >= *flagMaxFails) {
 				atomic.StoreInt32(&exitFlag, 1)
 			}
-			f, err := ioutil.TempFile("", "go-stress")
-			if err != nil {
-				fmt.Printf("failed to create temp file: %v\n", err)
-				os.Exit(1)
-			}
-			f.Write(out)
-			f.Close()
-			if len(out) > 2<<10 {
-				out = out[:2<<10]
+			if *flagStdErr {
+				fmt.Fprintf(os.Stderr, "\n%s\n", out)
```

Comment:
I'd use stdout instead of stderr for this. The errors from the program being tested are the primary output of this tool; if we're not going to preserve the original program's stdout/stderr distinction we gain nothing by writing everything to stderr here. (if anything the progress message that runs every 5 seconds should go to stderr while the logs go to stdout).